src/app/components/shell/router.js

```diff
@@ -133,39 +135,50 @@ function MainRoutes() {
         <Layout>
             <Routes>
                 <Route path="/" element={<ImportedPage name="home" />} />
-                {
-                    FOOTER_PAGES.map(
-                        (path) => <Route path={path} key={path} element={<ImportedPage name="footer-page" />} />
-                    )
-                }
-                <Route path="/errata/" element={<ImportedPage name="errata-summary" />} />
-                <Route path="/errata/form/" element={<ImportedPage name="errata-form" />} />
-                <Route path="/errata/*" element={<ImportedPage name="errata-detail" />} />
-                <Route path="/details/books/:title" element={<ImportedPage name="details" />} />
-                <Route path="/details/:title" element={<RedirectToCanonicalDetailsPage />} />
-                <Route path="/details/" element={<Navigate to="/subjects" replace />} />
-                <Route path="/books/:title" element={<RedirectToCanonicalDetailsPage />} />
-                <Route path="/textbooks/:title" element={<RedirectToCanonicalDetailsPage />} />
-                <Route path="/subjects/*" element={<ImportedPage name="subjects" />} />
-                <Route path="/k12/*" element={<ImportedPage name="k12" />} />
-                <Route path="/blog/*" element={<ImportedPage name="blog" />} />
```

Comment:
i'm wondering if there are any of these, like k12, blog, general, that we explicitly don't want to work in the portal. i guess probably they're not hurting anything by being in there